xds/src/main/java/io/grpc/xds/CsdsService.java

```diff
@@ -101,21 +106,31 @@ public void onCompleted() {
 
   private boolean handleRequest(
       ClientStatusRequest request, StreamObserver<ClientStatusResponse> responseObserver) {
+    StatusException error = null;
     try {
       responseObserver.onNext(getConfigDumpForRequest(request));
-      return true;
```

Comment:
Can keep `return true` now?

Reply:
No problem, moved to returning `true` from the `try`.